### PR TITLE
[tests-only] Follow renaming of konnectd to idp from ocis repo

### PR DESCRIPTION
**Pull request by kulmann**
Konnectd has been renamed to IdP in the ocis repo. This PR adjusts drone accordingly.

### CODE REVIEW

**Comment by phil-davis** [comment]
https://github.com/owncloud/ocis/pull/1566 should help with https://drone.owncloud.com/owncloud/web/12954/5/13
```
stat /srv/app/src/github.com/owncloud/ocis/idp/cmd/idp: directory not found
```

File: .drone.star [hunk 1]
```diff
@@ -1255,7 +1255,7 @@ def setupGraphapiOIdC():
 			'php occ a:e graphapi',
 			'php occ a:e openidconnect',
 			'php occ config:system:set trusted_domains 2 --value=web',
-			'php occ config:system:set openid-connect provider-url --value="https://konnectd:9130"',
+			'php occ config:system:set openid-connect provider-url --value="https://idp:9130"',
```
Comment (kulmann) [inline .drone.star:1258]: @phil-davis is it correct to just change the alias? Or is that stored somewhere magically, so that it has to stay `konnectd`?

Comment (phil-davis) [reply .drone.star:1258]: The name of the pipeline step should be the thing that translated to the address of the docker in which the processes of that step are running. So if the pipeline step is now called `idp` then it should be accessible there.

Comment (kulmann) [reply .drone.star:1258]: cool, thanks 👍

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/web/12955/79/11
```
runsh: There were no unexpected failures.
runsh: Total unexpected passed scenarios throughout the test run:
webUITrashbinDelete/trashbinDelete.feature:114
```

With the newer OCIS, that scenario is passing. So it needs to be removed from expected-failures.

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/web/12955/47/20
```
The following logs were found in the browser console:

4:02:55 PM - SEVERE - http://web:9100/core/core.bundle.js 17:102728 "JsonService.getJson: network error"
4:02:55 PM - SEVERE - https://konnectd:9130/.well-known/openid-configuration - Failed to load resource: net::ERR_NAME_NOT_RESOLVED
4:02:55 PM - SEVERE - http://web:9100/core/core.bundle.js 17:117408 Uncaught Error: Network Error
```

I guess that there are mentions of `konnectd` elsewhere that need changing...

**Comment by phil-davis** [comment]
I pushed 2 commits for the things that I could see. Let's see what CI  thinks.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**